vendors/halo2/src/bn254_shplonk_prover.cc

```diff
@@ -101,7 +101,12 @@ void SHPlonkProver::create_proof(ProvingKey& key,
 
   tachyon_halo2_bn254_argument_data* data =
       tachyon_halo2_bn254_argument_data_create(num_circuits);
+
```

Comment:
Is there a particular reason you added a empty line here?

Reply:
To put `reserve` and `add` together?